app/src/main/java/com/pwhs/quickmem/presentation/app/settings/about/open_source_licenses/detail/OpenSourceDetailScreen.kt

```
@@ -0,0 +1,107 @@
+                    append(license.linkSource)
+                    addStyle(
+                        style = SpanStyle(color = Color.Blue),
+                        start = 0,
+                        end = license.linkSource.length
+                    )
+                },
+                onClick = {
+                    val intent = Intent(Intent.ACTION_VIEW, Uri.parse(license.linkSource))
+                    try {
+                        context.startActivity(intent)
+                    } catch (e: ActivityNotFoundException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)